src/index.js

```diff
@@ -149,6 +149,12 @@ window.realityEditor = {
             vertexShader: {},
         }
     },
+    spatialCapture: {
+        CameraVisPatch: {},
```

Comment:
I don't see anything filling out these empty objects. It looks more like this should be `spatialCapture: {}` for the later `realityEditor.spatialCapture.spatialPatchCoordinator = new SpatialPatchCoordinator();`